### DISCUSSION TITLE
Dark Mode Emails

### DISCUSSION THREAD

**Discussion by ZakGhost354313** [dailydotdev/daily#680] [Feature Requests]
You should add an optional Dark Mode on Emails, I just got blinded.

**Comment by idoshamun** [comment]
@ZakGhost354313 I hope your eyes are ok now. 🥲
Let me check internally what it takes to support dark mode in our emails and I'll get back to you

**Reply by idoshamun** [reply]
@ZakGhost354313 I checked and we officially support dark mode in our emails if your device is set to dark mode. 
Can you please attach the problematic email and email client?
Is your device set to dark mode?
